src/strategies/coinswap/index.ts

```diff
@@ -0,0 +1,168 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+import examplesFile from './examples.json';
+
+export const author = 'CoinSwap-Space';
+export const version = '0.0.1';
+export const examples = examplesFile;
+
+const abi = [
+  'function balanceOf(address account) external view returns (uint256)'
+];
+
+const masterCSSAbi = [
+  'function userInfo(uint256, address) view returns (uint256 amount, uint256 rewardDebt)',
+  'function totalSupply() view returns (uint256)',
+  'function balanceOf(address _owner) view returns (uint256 balance)'
+];
+
+const communityStakeAbi = [
+  'function userInfo(address) view returns (uint256 amount, uint256 rewardDebt)'
+];
+
+const bn = (num: any): BigNumber => {
+  return BigNumber.from(num.toString());
+};
+
+const addUserBalance = (userBalances, user: string, balance) => {
+  if (userBalances[user]) {
+    return (userBalances[user] = userBalances[user].add(balance));
+  } else {
+    return (userBalances[user] = balance);
+  }
+};
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = 'latest';
```

Comment:
Hi, @PawelGozdz I just noticed that you are not using the `snapshot` block number to get the result here, and the voting power generated by this strategy will keep changing in the future, this is not expected behaviour for a strategy in snapshot

Please add a fix to this or this strategy will be disabled soon,

Thanks